.github/actionlint.yml

```diff
@@ -0,0 +1,16 @@
+# See https://github.com/rhysd/actionlint/blob/main/docs/config.md
```

Comment:
This was copied from E/App. It just defines our own self-hosted runners and includes a fix for a bug in actionslint. It will be necessary if we enable this workflow across all repos.

Reply:
done